packages/cli/bin/cli.js

```diff
@@ -74,6 +81,17 @@ const argv = yargs
     describe: 'set log level to debug',
     type: 'boolean',
   })
+  .check(argv => {
+    if (getIsInProject(argv.src)) {
+      throw new Error(
```

Comment:
The current pattern for logging errors is to use `logger.error` and then an explicit `process.exit` with an error code if you want to exit.